qiskit/providers/ibmq/experiment/experiment.py

```diff
@@ -68,6 +71,9 @@ def __init__(
                 group from the provider is used.
             project: The project to which this experiment belongs. If not specified the
                 project from the provider is used.
+            visibility: The visibility of the experiment which determines who can
+                see it when listing experiments.
```

Comment:
The current model is that a user can instantiate an `Experiment` class, meaning the docstring needs to be more descriptive (like what the valid values are).

Reply:
OK, I was following the minimal description for like AnalysisResult.quality.

This ties into my question above about the default, i.e. could someone mistakenly update the visibility because of the default on a newly constructed object without intentionally trying to do that just because they didn't got through the retrieve route first?

Reply:
`AnalysisResult.quality` values were not listed because they were mainly for the internal experimentalists and probably won't be used by general Qiskit users (values like "human bad" just sounds silly). I'm currently working on a new class that will have all the fields specifically for these experimentalists (like `fit`).